python_modules/dagster/dagster/check/__init__.py

```diff
@@ -850,6 +852,24 @@ def opt_str_param(obj: object, param_name: str, default: Optional[str] = None) -
     return default if obj is None else obj
 
 
+def path_param(obj: object, param_name: str) -> str:
```

Comment:
This should be put in its own section rather than in the string section.

Reply:
I may have just searched for opt_str_param and didn't try to read the whole file to see how it's broken up into sections.